packages/rx_bloc_list/lib/src/extensions/identifiable_extensions.dart

```diff
@@ -83,23 +82,23 @@ extension ModelManageEvents<E extends Identifiable> on Stream<E> {
   /// ```
   /// objectStream.withLatestFromIdentifiableList(
   ///     listStream,
-  ///     operationCallback: (updatedIdentifiable, identifiableInList) async => ManageOperation.remove,
+  ///     operationCallback: (updatedIdentifiable) async => ManageOperation.remove,
```

Comment:
Please add information in the change log, also please bump the version of the rx_bloc_list

Reply:
I added information about the change of this method `withLatestFromIdentifiableList  `and the other newly added method `withLatestFromIdentifiablePairList` and class `IdentifiablePair` and bumped the version in the pubspec.yaml